Makefile

```diff
@@ -1,9 +1,18 @@
-help = sed -n "/^$1/ { x ; p ; } ; s/\#\#/[⚙]/ ; s/\./.../ ; x" ${MAKEFILE_LIST}
+info = $$(tput setaf 6)[i]$$(tput sgr0)
+warn = $$(tput setaf 3)[!]$$(tput sgr0)
+work = $$(tput setaf 5)[⚙]$$(tput sgr0)
+pass = echo $$(tput setaf 2)[✓]$$(tput sgr0) Good work! =\> $$(tput setaf 8)$1$$(tput sgr0)$$(tput setaf 2)passed$$(tput sgr0) $$(tput setaf 1)❤$$(tput sgr0)
+help = sed -n "/^$1/ { x ; p ; } ; s/\#\#/$(work)/ ; s/\./.../ ; x" ${MAKEFILE_LIST}
```

Comment:
We at least need a comment on this incantation.

Reply:
I'm not very proud myself, but it works! Sure.